test/alternator/test_batch.py

```diff
@@ -375,6 +379,43 @@ def test_batch_write_item_large(test_table_sn):
     assert full_query(test_table_sn, KeyConditionExpression='p=:p', ExpressionAttributeValues={':p': p}
         ) == [{'p': p, 'c': i, 'content': long_content} for i in range(25)]
 
+# Test if client breaking connection during HTTP response
+# streaming doesn't break the server.
+def test_batch_write_item_large_broken_connection(test_table_sn, request, dynamodb):
+    fn_name = sys._getframe().f_code.co_name
+    ses = new_dynamodb_session(request, dynamodb)
+    # This disruption doesn't always work so we repeat it.
+    for _ in range(1, 20):
+        p = random_string()
+        long_content = random_string(100)*500
+        write_reply = test_table_sn.meta.client.batch_write_item(RequestItems = {
```

Comment:
By the way, is writing the same item again and again 20 times part of this test, or it's enough to just read it 20 times?

Reply:
good point, only read is required, will limit the loop